Bash_scripts/HW20/roles/hw201/tasks/docker.yml

```diff
@@ -0,0 +1,43 @@
+#Ubuntu
+- name: "Install Docker on Ubuntu"
+  block:
+    - name: "Set up the repository"
+      ansible.builtin.package:
+       name: "{{item}}"
+       update_cache: yes
+      loop:
+       - apt-transport-https
+       - ca-certificates
+       - gnupg
+       - libcurl4
+       - lsb-release
+
+    - name: "Add GPG key"
+      ansible.builtin.apt_key:
+       url: https://download.docker.com/linux/ubuntu/gpg
+       state: present
+       
+    - name: "Add repos"
+      ansible.builtin.apt_repository:
+       repo: deb https://download.docker.com/linux/ubuntu focal stable
+       state: present
+       
+    - name: Install and Start docker engine
```

Comment:
I think it make sense to use handler here to start docker. Docker install should be separated

Reply:
added handler to start docker